content/getting-started.md

```diff
@@ -1,22 +1,75 @@
 title: Getting Started
 h1: Getting started with complate
+safe: false
 
-Pattern Libraries
------------------
+<style>
+/* XXX: temporary workaround */
 
-We often develop our complate components using a pattern library.
-The following pattern libraries support complate:
+.diagram-complate-core {
+    max-width: 100%;
+    margin-bottom: 2rem;
```

Comment:
```
margin: 2rem auto;
display: block;
```

When we use margin:auto + display:block, the image is centered within the viewport. 
I would also move this rule into its own component CSS class (or just directly in the index.scss for now). The hack with the inline styles in the other file was because the CSS rule that was defined there could influence images on other pages, but this is correctly scoped with a class, so it should be fine.